pkg/volume/portworx/portworx_util.go

```diff
@@ -56,11 +56,14 @@ func (util *PortworxVolumeUtil) CreateVolume(p *portworxVolumeProvisioner) (stri
 	requestGB := int(volume.RoundUpSize(capacity.Value(), 1024*1024*1024))
 
 	specHandler := osdspec.NewSpecHandler()
-	spec, err := specHandler.SpecFromOpts(p.options.Parameters)
-	if err != nil {
-		glog.Errorf("Error parsing parameters for PVC: %v. Err: %v", p.options.PVC.Name, err)
-		return "", 0, nil, err
+	spec, _ := specHandler.SpecFromOpts(p.options.Parameters)
+	if spec == nil {
+		spec = specHandler.DefaultSpec()
```

Comment:
I am puzzled by this PR. StorageClass A has these parameters:

```
   repl: "3"
```

`SpecFromOpts` returns a spec with `HaLevel = 3` and you pass it to `driver.Create`. Cool.

StorageClass B has these parameters:

```
   repl: "3"
   io_priority: "high"
```

In #49526 you say that `SpecFromOpts` fails. Now you pass the default spec with `HaLevel = 1, Cos = 0` and with `repl` and `io_priority` labels to `driver.Create`. Is this correct? Will `driver.Create` parse the labels again and override `spec.HaLevel` and `spec.Cos`? So what's the point of calling `SpecFromOpts` if everything can be passed as labels?